stdlib/runtime/wasi.gr

```diff
@@ -77,10 +77,10 @@ provide foreign wasm fd_prestat_dir_name: (
 /**
  * Invokes the `fd_write` system call.
  *
- * @param 0: The file descriptor to write to
- * @param 1: The pointer to the array of iovs to write
- * @param 2: The length of the array of iovs
- * @param 3: Where to store the number of bytes written
+ * @param 1: The file descriptor to write to
+ * @param 2: The pointer to the array of iovs to write
+ * @param 3: The length of the array of iovs
+ * @param 4: Where to store the number of bytes written
```

Comment:
The general idea of this change sounds reasonable, but when I read this it makes me feel like it should remain 0-based. What are other cases where this will show up?